CONTRIBUTING.rst

```diff
@@ -6,7 +6,7 @@ CLI Helpers would love your help! We appreciate your time and always give credit
 Development Setup
 -----------------
 
-Read to contribute? Here's how to set up CLI Helpers for local development.
+Ready to contribute? Here's how to set up CLI Helpers for local development.
```

Comment:
below `cli_helper_dev` should be renamed to `cli_helpers_dev`, besides that, I get errors when the venv is in the `cli_helpers_dev` directory, I need to place it a directory above it.